numpy/typing.pyi

```diff
@@ -0,0 +1,64 @@
+import sys
```

Comment:
Copy paste of https://github.com/numpy/numpy-stubs/blob/master/numpy-stubs/typing.pyi.

Next step is to make this available at runtime too.